<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditOfferDialog</class>
 <widget class="QDialog" name="EditOfferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Offer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="aliasLabel">
       <property name="text">
        <string>Seller Alias:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="aliasEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="offerLabel">
       <property name="text">
        <string>Offer:</string>
       </property>
       <property name="buddy">
        <cstring>offerEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="offerEdit">
       <property name="toolTip">
        <string>The value associated with this certificate.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Title:</string>
       </property>
       <property name="buddy">
        <cstring>nameEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="nameEdit">
       <property name="toolTip">
        <string>The certificate name.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="categoryEdit"/>
     </item>
     <item row="9" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="priceEdit"/>
     </item>
     <item row="11" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="qtyLabel">
       <property name="text">
        <string>Quantity:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="qtyEdit"/>
     </item>
     <item row="13" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="currencyLabel">
       <property name="text">
        <string>Currency:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QComboBox" name="currencyEdit">
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="currencyDisclaimer">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="certLabel">
       <property name="text">
        <string>Certificate:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QComboBox" name="certEdit"/>
     </item>
     <item row="17" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_8"/>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QPlainTextEdit" name="descriptionEdit">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_10"/>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Private:</string>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QComboBox" name="privateEdit"/>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="aliasDisclaimer">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Accept BTC Only:</string>
       </property>
      </widget>
     </item>
     <item row="24" column="1">
      <widget class="QComboBox" name="acceptBTCOnlyEdit"/>
     </item>
     <item row="25" column="1">
      <widget class="QLabel" name="btcOnlyDisclaimer">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <widget class="QLabel" name="privateDisclaimer">
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Alias Peg:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="aliasPegEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="aliasPegDisclaimer">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditOfferDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditOfferDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
